src/sponsor.html

```diff
@@ -2,11 +2,13 @@
 <html lang="en">
 
 <head>
+  <meta http-equiv="refresh" content="0; URL='https://mchacks.ca/static/sponsor.pdf'" />
```

Comment:
This solution is not ideal as it requires the use of the mchacks.ca domain and is not portable. A better solution is to us an `.htaccess` redirect that is independent of the domain it's being served from.